src/rpc/RPC.cpp

```diff
@@ -91,6 +91,37 @@ make_HttpContext(
         clientIp};
 }
 
+constexpr static WarningInfo warningInfos[]{
+    {warnUNKNOWN, "Unknown warning"},
+    {warnRPC_CLIO,
+     "This is a clio server. clio only serves validated data. If you "
+     "want to talk to rippled, include 'ledger_index':'current' in your "
+     "request"},
+    {warnRPC_OUTDATED, "This server may be out of date"}};
```

Comment:
While we are here, can we add a proper warning for when clients are about to be rate limited? I know we have the string `load` in a separate field, and we can leave that, but I think it would be easier for clients to be able to parse all warnings in one place.

Reply:
Sounds good, I'll add that